clusterloader2/pkg/measurement/common/metrics/metrics_grabber.go

```diff
@@ -192,27 +190,6 @@ func (g *Grabber) GrabFromControllerManager() (ControllerManagerMetrics, error)
 		return ControllerManagerMetrics{}, fmt.Errorf("master's Kubelet is not registered. Skipping ControllerManager's metrics gathering")
 	}
 
-	var err error
```

Comment:
We didn't have anything like that in kube-scheduler or in any other components.

To fully eliminate dependency on test/e2e from k/k I'm taking the simplest approach of removing it here - to unblock https://github.com/kubernetes/perf-tests/pull/3050

Reply:
SG